packages/bsky/src/context.ts

```diff
@@ -52,6 +53,10 @@ export class AppContext {
     return this.opts.suggestionsAgent
   }
 
+  get topicsAgent(): AtpAgent | undefined {
```

Comment:
is this going out to Discover? tempted to just re-use the suggestionsAgent in the meantime, unless we have a reason to introduce a new one

not very concerned either way though. Happy to keep it as is as well

Reply:
have it on a separate box rn which is why i created the separate agent. we might combine it sooner than later at which point we could def drop this.